pmoves/services/flute-gateway/main.py

```
@@ -1003,6 +1051,201 @@ async def get_persona(persona_id: str) -> Dict[str, Any]:
+            )
+
+    except ValueError as exc:
+        REQUESTS_TOTAL.labels(endpoint="/v1/voice/clone/synthesize", status="400").inc()
+        raise HTTPException(status_code=400, detail=str(exc)) from exc
+    except NotImplementedError as exc:
+        REQUESTS_TOTAL.labels(endpoint="/v1/voice/clone/synthesize", status="501").inc()
+        raise HTTPException(status_code=501, detail=str(exc)) from exc
+    except Exception:
+        REQUESTS_TOTAL.labels(endpoint="/v1/voice/clone/synthesize", status="500").inc()
+        logger.exception("Cloned voice synthesis failed")
+        raise HTTPException(status_code=500, detail="Cloned voice synthesis failed")
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)